README.rst

```diff
@@ -54,197 +59,107 @@ All-in-one extraction
-                               'geo': {'@type': 'GeoCoordinates',
-                                       'latitude': 40.660109,
-                                       'longitude': -73.953193},
-                               'name': 'The Owl Music Parlor',
-                               'sameAs': 'http://www.theowl.nyc'},
-                  'name': 'Elysian Fields',
-                  'performer': [{'@type': 'MusicGroup',
-                                 'name': 'Elysian Fields',
-                                 'sameAs': 'http://www.songkick.com/artists/236156-elysian-fields?utm_medium=organic&utm_source=microformat'}],
-                  'startDate': '2017-06-10T19:30:00-0400',
-                  'url': 'http://www.songkick.com/concerts/30173984-elysian-fields-at-owl-music-parlor?utm_medium=organic&utm_source=microformat'},
-                 (...)
-                 {'@context': 'http://schema.org',
-                  '@type': 'MusicGroup',
-                  'image': 'https://images.sk-static.com/images/media/profile_images/artists/236156/card_avatar',
-                  'interactionCount': '5557 UserLikes',
-                  'logo': 'https://images.sk-static.com/images/media/profile_images/artists/236156/card_avatar',
-                  'name': 'Elysian Fields',
-                  'url': 'http://www.songkick.com/artists/236156-elysian-fields?utm_medium=organic&utm_source=microformat'}],
-     'microdata': [],
-     'rdfa': [{'@id': 'http://www.songkick.com/artists/236156-elysian-fields',
-               'al:ios:app_name': [{'@value': 'Songkick Concerts'}],
-               'al:ios:app_store_id': [{'@value': '438690886'}],
-               'al:ios:url': [{'@value': 'songkick://artists/236156-elysian-fields'}],
-               'http://ogp.me/ns#description': [{'@value': 'Buy tickets for an '
-                                                           'upcoming Elysian '
-                                                           'Fields concert near '
-                                                           'you. List of all '
-                                                           'Elysian Fields tickets '
-                                                           'and tour dates for '
-                                                           '2017.'}],
-               'http://ogp.me/ns#image': [{'@value': 'http://images.sk-static.com/images/media/img/col4/20100330-103600-169450.jpg'}],
-               'http://ogp.me/ns#site_name': [{'@value': 'Songkick'}],
-               'http://ogp.me/ns#title': [{'@value': 'Elysian Fields'}],
-               'http://ogp.me/ns#type': [{'@value': 'songkick-concerts:artist'}],
-               'http://ogp.me/ns#url': [{'@value': 'http://www.songkick.com/artists/236156-elysian-fields'}],
-               'http://www.facebook.com/2008/fbmlapp_id': [{'@value': '308540029359'}]}]}
-
-
-You can also use each extractor individually. See below.
-
-Microdata extraction
-++++++++++++++++++++
```

Comment:
Why is this section removed?

Reply:
I am still modifying the readme, I committed it by mistake

Reply:
I had to remove because by adding microformat and opengraph the output was different and I found it easier to copy paste the new version instead of modifying the single lines